lib/Doctrine/Table.php

```diff
@@ -1179,6 +1179,10 @@ public function getColumnName($fieldName)
             return $this->_columnNames[$fieldName];
         }
 
+        if (is_null($fieldName)) {
```

Comment:
What about `if (null === $fieldName)`
wdyt?

Just for the sake of being forward-compatible with the cs-fixer (still working on it) :)

Reply:
No problem, I've updated the pr.